querybook/webapp/components/ListDataDoc/components/ActiveButtons.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { useState } from 'react';
```

Comment:
for the sake of standardization, don't put it in to a components/ folder and leave it in ListDataDoc/